src/backend.github.js

```diff
@@ -217,7 +217,12 @@ let _ = Mavo.Backend.register($.Class({
 
 							this.repoInfo = repoInfo;
 
-							if (!this.mavo.source) { // if url doesn't have source, check for forks
+							if (this.mavo.storage) { // Check for forks
+								if (repoInfo.fork) { // Current repo is a fork
```

Comment:
I removed this check from the code by mistake when I was moving the code concerning PRs to the plugin. I returned it since it helps to avoid some extra unneeded checks.